### ISSUE TITLE
Project/library name

### ISSUE THREAD

**Issue by Necr0x0Der** [trueagi-io/metta-morph#14]
`metta-morph` sounds nice, and I don't argue for changing its name, but we may need to just fix the decision.

* As for `metta-` prefix: `import metta-something` in MeTTa looks redundant. However, it's ok to have it for the repo or even package name. E.g., `pytorch` is a package name, while it is imported as `torch`. MeTTa package installation is not ready, so it is more a question about the repo name. `metta-` sounds suitable for it.

* As for `morph`, it is not too discriminative and descriptive. It says nothing about Scheme or compilation. However, it may become a library for transforming metta programs (in a meta-programming style), who knows. Thus, it is not completely out of place.

In total, `metta-morph` is OK, but if it was considered as a preliminary working name and there are any ideas for renaming, then it is timely to consider them or decide to stay with the current name.

**Comment by patham9** [comment]
I agree, and initially I picked the name since overall it "morphs" the MeTTa code to become Scheme then C then binary, which has some loose analogy with Metamorphosis in biology.

**Comment by ntoxeg** [comment]
Just FYI, Pytorch’s package name **is** `torch` these days. “PyTorch” is only the official project name at this point. Some other libraries also have adopted such transition, like `pytorch-lightning` -> `lightning`. Under that convention your package would just be named `morph`. Whether that’s ambiguous or not depends on how standalone the project is supposed to be, otherwise maybe it’s better to clarify that it’s some MeTTa-related toolkit.
